uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/apis/leave-request-api.js

```diff
@@ -240,6 +241,67 @@ define([
           .then(function (response) {
             return response.values;
           });
+      },
+
+      /**
+       * Calls the getcomment backend API.
+       *
+       * @param {String} leaveRequestID - Id of leave request
+       * @param {Object} params
+       *
+       * @return {Promise}
+       */
+      getComments: function (leaveRequestID, params) {
+        params = _.assign(params, {
+          leave_request_id: leaveRequestID
+        });
+
+        return this.sendGET('LeaveRequest', 'getcomment', params)
```

Comment:
any custom methods are generally POST, can you check if this works in civi explorer?